### ISSUE TITLE
Formatting is not working on windows.

### ISSUE THREAD

**Issue by saddamhshovon** [open-southeners/vscode-laravel-pint#52]
v1.1.6 is not formatting on widows system. but v1.1.4 is working properly.

**Comment by d8vjork** [comment]
Ok need an extra eyes here, as I don't use Windows

**Comment by saddamhshovon** [comment]
please let me know if i can help in any way

**Comment by medilies** [comment]
I face the same issue for a week.

- Environment: windows 10
- Extension version: 1.1.6
- Pint: 1.13.7

```shell
code --version
1.85.1
0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
x64
```

**Comment by clementmas** [comment]
Thanks for reporting this issue. I'm downgrading to 1.1.4 until it is fixed.

**Comment by d8vjork** [comment]
This must be fixed with #55
